### DISCUSSION TITLE
Is `testbench make:test` supposed to work?

### DISCUSSION THREAD

**Discussion by geoffreyvanwyk** [orchestral/testbench#407] [Q&A] [answered]
When I run `testbench make:test` and answer the prompts, I get this error message:

```
  BadMethodCallException

  Generating test is not supported for [workbench] preset

  at vendor/orchestra/workbench/src/GeneratorPreset.php:167
    163▕      * @throws \BadMethodCallException
    164▕      */
    165▕     public function testingNamespace()
    166▕     {
  ➜ 167▕         throw new BadMethodCallException('Generating test is not supported for [workbench] preset');
    168▕     }
    169▕
    170▕     /**
    171▕      * Preset has custom stub path.
```

Is this command supposed to work?

**Comment by crynobone** [comment, marked as answer]
No, `workbench` folder doesn't have a tests directory.

**Reply by geoffreyvanwyk** [reply]
OK. Thanks.

The `workbench` directory gets committed with the rest of the package. Would it make sense to rather have the `tests` directory in there, instead of in the package root?

**Reply by crynobone** [reply]
No, `workbench` can be used for tests. Still, its main feature is the ability to preview/interact with the packages via CLI or browser without requiring it inside a Laravel application.
